war/src/main/scss/abstracts/_theme.scss

```diff
@@ -174,8 +174,8 @@ $semantics: (
   --table-header-foreground: var(--text-color);
   --table-body-background: var(--background);
   --table-body-foreground: var(--text-color);
-  --table-border-radius: 10px;
-  --table-row-border-radius: 4px;
+  --table-border-radius: 0.75rem;
+  --table-row-border-radius: 0.3125rem;
```

Comment:
What's the unit change here for? <s>Weirdos</s> People who "zoom text only" on web pages?

Reply:
This explains it well: https://stackoverflow.com/a/43131958

Generally in new web development `rem` is used pretty much exclusively (mostly) as it adapts to the users font-size

Reply:
Could you be more explicit here? Because there's no difference for full-page zoom; only font-only zoom behaves different with this change.